using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -854,18 +963,23 @@ ACPCore.setPushIdentifier(deviceToken)
 {% tab title="React Native" %}
 #### JavaScript
 
-### setPushIdentifier
+### setPushIdentifier<a id="setPushIdentifier-js"></a>
 
 ```jsx
 ACPCore.setPushIdentifier("pushIdentifier");
 ```
+o *pushIdentifier* is a String that contains the device token for push notifications.
```

Comment:
Reword to:
"* *pushIdentifier* is a string that contains the device token for push notifications."